MilitaryElite/MilitaryElite/Other/Mission.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MilitaryElite.Other
+{
+    internal class Mission
+    {
+        public string CodeName { get; }
+        public string State { get; private set; }
+
+        public Mission(string codeName, string state)
+        {
+            CodeName = codeName;
+            State = state;
+        }
+
+        public void Complete()
+        {
+            State = "Finished";
```

Comment:
Better to have enumeration for different mission states. What will happen if the state is already 'Finished'